src/main/java/com/spotinst/sdkjava/example/statefulnode/azure/AzureStatefulNodeExample.java

```diff
@@ -0,0 +1,558 @@
+        StatefulNodeTasksConfiguration task2 =
+                tasksBuilder2.setIsEnabled(false).setCronExpression("25 * * * *").setType(AzureStatefulNodeTaskTypeEnum.PAUSE).build();
+        List<StatefulNodeTasksConfiguration> tasksList = new ArrayList<>();
+        tasksList.add(task1);
+        tasksList.add(task2);
+
+        //Build Scheduling
+        StatefulNodeSchedulingConfiguration scheduling =
+                StatefulNodeSchedulingConfiguration.Builder.get().setTasks(tasksList).build();
+
+        StatefulNode.Builder statefulNodeBuilder = StatefulNode.Builder.get();
+        StatefulNode statefulNodeToUpdate = statefulNodeBuilder.setScheduling(scheduling).build();
+
+        // Build node creation request
+        StatefulNodeCreationRequest.Builder nodeCreationRequestBuilder = StatefulNodeCreationRequest.Builder.get();
+        StatefulNodeCreationRequest updationRequest = nodeCreationRequestBuilder.setNode(statefulNodeToUpdate).build();
+
+        // Convert node to API object json
+        System.out.println(String.format("Update Request for stateful Node: %s ", nodeId));
+        System.out.println(updationRequest.toJson());
+
+        StatefulNode updateNodeResponse = client.updateNode(updationRequest,nodeId);
+        System.out.println("Update Stateful Node Scheduling Tasks is successful");
+
+        return updateNodeResponse;
+
+    }
+
+    private static StatefulNode updateStatefulNodeHealth(SpotinstAzureStatefulNodeClient client, String nodeId) {
+
+        //Build Health
+        List<HealthCheckTypeEnumAzure> healthCheckTypesList = new ArrayList<>();
+        healthCheckTypesList.add(HealthCheckTypeEnumAzure.VM_STATE);
+
+        StatefulNodeHealthConfiguration.Builder healthBuilder = StatefulNodeHealthConfiguration.Builder.get();
+        StatefulNodeHealthConfiguration health =
+                healthBuilder.setHealthCheckTypes(healthCheckTypesList).setUnhealthyDuration(200).setGracePeriod(240).setAutoHealing(false).build();
+
+        StatefulNode.Builder statefulNodeBuilder = StatefulNode.Builder.get();
+        StatefulNode statefulNodeToUpdate = statefulNodeBuilder.setHealth(health).build();
+
+        // Build node creation request
+        StatefulNodeCreationRequest.Builder nodeCreationRequestBuilder = StatefulNodeCreationRequest.Builder.get();
+        StatefulNodeCreationRequest updationRequest = nodeCreationRequestBuilder.setNode(statefulNodeToUpdate).build();
+
+        // Convert node to API object json
+        System.out.println(String.format("Update Request for stateful Node: %s ", nodeId));
+        System.out.println(updationRequest.toJson());
+
+        StatefulNode updateNodeResponse = client.updateNode(updationRequest,nodeId);
+        System.out.println("Update Stateful Node Health is successful");
+
+        return updateNodeResponse;
+
+    }
+
+    private static StatefulNode updateStatefulNodePersistent(SpotinstAzureStatefulNodeClient client, String nodeId) {
```

Comment:
Change function name to updateStatefulNodePersistence

Reply:
Done